### PR TITLE
Mbtiles reader

### PR DESCRIPTION
**Pull request by ldgeo**
this PR adds the support for MBTiles as it's the main format used in openmaptiles to serve tiles from files and a common tileset's format to use in Mapbox Studio.

### CODE REVIEW

**Comment by ldgeo** [comment]
rebased on master ping @autra

File: README.md [hunk 1]
```diff
@@ -41,5 +40,12 @@ Then layer `boundaries` can be served with:
 
 `fields` is optional, and when absent only geometries are encoded in the vector tile.
 
+## Preview 
+
+When the source is a tm2 file, the root endpoint will display a built-in viewer with `mapbox-gl-js`.
```
Comment (autra) [inline README.md]: You also have a viewer when pointing to a mbtiles right?

Comment (autra) [reply README.md]: Thinking out loud (for another PR): when using a direct connexion to postgis, the index.html could display a form asking for the layer name to display, before we can implement our mechanism for discovering postgis layer.

**Review by autra — CHANGES_REQUESTED** [review]
Just a little nits on a comment, and it's good to go!

File: README.md (hunk 1 above)
Comment (ldgeo) [reply README.md]: > You also have a viewer when pointing to a mbtiles right?

Fixed !

Comment (ldgeo) [reply README.md]: > Thinking out loud (for another PR): when using a direct connexion to postgis, the index.html could display a form asking for the layer name to display, before we can implement our mechanism for discovering postgis layer.

For that we also need to apply a ramdom style dynamically.

Comment (autra) [reply README.md]: I think I'm missing something here: we can't apply - for instance - the style.json in Oslandia/postile-openmaptiles?

**Review by autra — APPROVED** [review]
Thanks! Do you ned to squash some commits?

File: README.md (hunk 1 above)
Comment (ldgeo) [reply README.md]: you said "when using a direct connexion to postgis" which is a quite different use case than `postile-openmaptiles`.  

The style.json is suited for the openmapstyles schema only.

**Comment by ldgeo** [comment]
@autra, just squashed, thanks for the review

**Pull request merged**